src/main/java/com/github/webicitybrowser/webicity/renderer/backend/html/tags/TagAction.java

```diff
@@ -0,0 +1,21 @@
+package com.github.webicitybrowser.webicity.renderer.backend.html.tags;
+
+import com.github.webicitybrowser.spec.css.rule.CSSRuleList;
+import com.github.webicitybrowser.spec.css.stylesheet.CSSStyleSheet;
+import com.github.webicitybrowser.spec.dom.node.Element;
+import com.github.webicitybrowser.spec.html.node.HTMLDocument;
+
+
+public interface TagAction {
+
+	CSSRuleList getCSSRuleList(Element element);
+
+	default void onTagParsed(Element element) {
```

Comment:
Oh, I meant the interface's default method should have an empty body
The logic below should not be located in this interface, as this interface will also be used for things not styling-related.
I would suggest moving it to to CSSRulesUtils

Reply:
so if this interface will be used for things not styling related, maybe i should get rid of getCSSRuleList method and rather do something like handleTag(Element element) and there add stylesheet. 
You will se what i mean in next commit